eclipse-jdt/eclipse.jdt.ui
# [WIP] adapt to renaming `UnnamedClass` to `ImplicitTypeDeclaration`

Signed-off-by: David Thompson <davthomp@redhat.com>

## What it does
Adapts to changes in https://github.com/eclipse-jdt/eclipse.jdt.core/pull/2220

## How to test
Ensure that the code compiles and tests pass when using this PR with the next I-Build of `org.eclipse.jdt.core`

## Author checklist
